package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vsc-extension-ansible",
   "displayName": "Ansible",
   "description": "VSCode extension for Ansible",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
we should release from 0.0.1, right?

Reply:
well, doesn't matter, but i reverted anyway. i just thought i had to increment when installing next vsix version locally, but it doesnt' matter.